crossdock/services/tracehandler.go

```diff
@@ -129,10 +129,7 @@ func (h *TraceHandler) runTest(service string, request *traceRequest, tFunc test
 	if err != nil {
 		return err
 	}
-	if err := vFunc(request, traces); err != nil {
-		return err
-	}
-	return nil
+	return vFunc(request, traces)
```

Comment:
These seem to be changes unrelated to this PR.

Reply:
Because go vet complained